repository/config_runtime.go

```diff
@@ -0,0 +1,59 @@
+package repository
+
+import (
+	"strconv"
+	"strings"
+)
+
+type runtimeConfig struct {
```

Comment:
something like `type runtimeConfig map[string]string` could also be correct and used directly in the `mockRepoForTest`